mix.exs

```diff
@@ -55,10 +55,11 @@ defmodule PhoenixSwagger.Mixfile do
   defp deps do
     [
       {:poison, "~> 2.2 or ~> 3.0"},
```

Comment:
Can the Poison dependency be made optional?

Reply:
@mbuhot yes, I think that's a good idea. would you like me to make the change?

Reply:
Yes please. 👍

Reply:
here we go...